app/components/ch/components/DownloadList.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <li>
+    <button
+      class="download-item"
+      :href="url"
+      target="_blank"
+      :download="title"
+      :id="`download-item-${id}`"
+    >
+      <SvgIcon icon="Download" size="xl" class="download-item-icon" />
+      <div>
+        <p class="download-item-title">
+          {{ title }}
+        </p>
+        <p v-if="description" class="download-item-description">
+          {{ description }}
+        </p>
+        <p class="download-meta-info">
```

Comment:
What about something more generic, like `meta-info` ? We may reuse this somewhere else ?

Reply:
yep. done